### ISSUE TITLE
Not able to move the coordinates tooltips of the map using Tab/arrow keys

### ISSUE THREAD

**Issue by yangster23** [WiredSolutions/react-azure-maps#85]
React azure maps fails  [SC 2.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html) accessibility requirement. 

User is not able to move the coordinates tooltips of the map using Tab/arrow keys. When we move to the map we are able to see only one tooltip data and not able to move other tooltips.

As a result, keyboard only users will not be able to view the coordinates tooltip data.

**Comment by rbrundritt** [comment]
I believe this is an implementation issue, not a bug in this SDK. The accessible popup example you reference creates a popup for each location. You will need to do the same in your app rather than reusing a single popup instance.
